microsoft/testsuites/performance/common.py

```diff
@@ -84,7 +86,9 @@ def perf_disk(
     # fail with EAGAIN.
     # read: https://www.kernel.org/doc/Documentation/sysctl/fs.txt
     # So we set numjob to 256 if numjob is larger than 256.
-    numjob = min(numjob, 256)
+    # This limitation is only needed for 'libaio' ioengine but not for 'io_uring'.
+    if ioengine == "libaio":
```

Comment:
Use an enum for the ioengine types to avoid typo.

Reply:
Added 3 new constants for io engines in "lisa\lisa\util\constants.py"

Reply:
Constants reduce typos for the string. The enum limit options of this parameter.

Reply:
Changed the logic to use enum